### PR TITLE
Add Execution Environments into a few screens

### PR DESCRIPTION
**Pull request by nixocio**
Add EE to the following screens:

* Job Template
* Organization
* Project
* Workflow Job Template

Also, add a new lookup component - ExecutionEnvironmentLoookup.

See: https://github.com/ansible/awx/issues/9189

### CODE REVIEW

**Comment by nixocio** [comment]
hey @jakemcdermott, I will address inventory source in another PR.

**Comment by jakemcdermott** [comment]
@nixocio I see some extra commits in the PR -  It might need a rebase or a different base branch.

**Review by jakemcdermott — CHANGES_REQUESTED** [review]
@nixocio I see some extra commits in the PR - It might need a rebase or a different base branch.

File: awx/ui_next/src/screens/Template/shared/JobTemplateForm.jsx [hunk 1]
```diff
@@ -270,6 +306,26 @@ function JobTemplateForm({
...
+          onChange={value => executionEnvironmentHelpers.setValue(value)}
+          popoverContent={i18n._(
+            t`Select the execution environment for this job template.`
+          )}
+          tooltip={i18n._(
+            t`Select a project before editing the execution environment.`
+          )}
+          globallyAvailable
+          isDisabled={!projectField.value}
+          organizationId={projectData?.organization}
```
Comment (jakemcdermott) [inline awx/ui_next/src/screens/Template/shared/JobTemplateForm.jsx:326]: Nice!

**Review by jakemcdermott — APPROVED** [review]

**Pull request merged**